numpy_cpp-custom-matrix/test.py

```diff
@@ -26,3 +26,12 @@
 print("A = ", A)
 print("B = ", B)
 print("A * B = ", example.Multiply(A,B,C,2,2,3))
+
+print()
+A = np.array([[1,1,1],[2,2,2]])
+C = np.array([[0,0],[0,0],[0,0]])
+print("A = ", A)
+print("A Transpose = ", example.Transpose(A,C,2,3))
```

Comment:
As I commented on Transpose() in example.cpp, 'A' will be the only argument passed.